<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Admin-PC" name="testcases.TC004_CreateLead" tests="2" failures="0" timestamp="14 Aug 2016 12:54:34 GMT" time="73.527" errors="1">
  <testcase name="login" time="49.132" classname="testcases.TC004_CreateLead"/>
  <testcase name="login" time="24.395" classname="testcases.TC004_CreateLead">
    <error type="java.lang.RuntimeException" message="FAILED">
      <![CDATA[java.lang.RuntimeException: FAILED
	at utils.Reporter.reportStep(Reporter.java:38)
	at wrappers.GenericWrappers.verifyTextContainsByXpath(GenericWrappers.java:228)
	at pages.HomePage.verifyUserName(HomePage.java:14)
	at testcases.TC004_CreateLead.login(TC004_CreateLead.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:744)
]]>
    </error>
  </testcase> <!-- login -->
</testsuite> <!-- testcases.TC004_CreateLead -->
